components/os/unix.go

```diff
@@ -29,6 +29,10 @@ func (*Unix) GetType() Type {
 	return UnixType
 }
 
+func (*Unix) GetRunAgentCmd(parameters string) string {
+	return "sudo datadog-agent " + parameters
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
Should we have sudo and non sudo linux commands ? This one might not work on debian, out of the box

Reply:
It works on debian.

Reply:
Ah ! I remember seeing a difference among Linux OS in kitchen tests, let's merge it as is and iterate later if needed, great PR !